src/lib/holocene/date-picker.svelte

```diff
@@ -89,7 +89,7 @@
   />
   {#if showDatePicker}
     <div
-      class="surface-primary absolute z-30 inline-block rounded border shadow"
+      class="surface-primary absolute z-30 inline-block rounded border border-subtle shadow dark:bg-[#101828]"
```

Comment:
Is `#101828` a new color? If so, maybe something we want to add to `/theme/plugins.ts` instead of having as a one-off here. 

Otherwise maybe there's an existing black we can use here instead?

Reply:
This is the 'main' background color for dark mode -- it only exists in one other place right now. This is the color from the designs. 

Do you have an opinion here for either switching to our black and checking with Ash or adding to plugin file? I truly don't know what the right move here would be (thus the hard coding) and would love guidance.

Reply:
I think we don't want any one-off colors and should add this to the plugin file. If there's not a noticeable difference between this and an existing black than I think it would be fine to just use the existing.